lib/frizz/ignorance.rb

```diff
@@ -2,13 +2,30 @@ module Frizz
   class Ignorance
     attr_reader :patterns
 
-    def initialize(patterns)
+    def initialize(patterns, prefer_gzip)
       @patterns = patterns || []
+      @prefer_gzip = prefer_gzip
     end
 
-    def ignore?(path)
+    def ignore?(local_path, full_path)
+      ignore_matched_pattern?(local_path) || ignore_for_gzip_version?(full_path)
+    end
+
+    private
+
+    def ignore_matched_pattern?(local_path)
       return false unless patterns.count
-      patterns.any? { |p| ::File.fnmatch(p, path) }
+
+      patterns.any? { |p| ::File.fnmatch(p, local_path) }
+    end
+
+    def ignore_for_gzip_version?(full_path)
+      return false unless @prefer_gzip
```

Comment:
@patbenatar So when I don't prefix the `prefer_gzip` instance variable with an `@`, i get an undefined variable/method error. However, I see that `patterns` is used with using the `@` symbol.

Any idea why this might happening?